terraform/environment/load_balancer_front.tf

```diff
@@ -130,15 +129,15 @@ resource "aws_lb_listener_rule" "www_redirect" {
     type = "redirect"
 
     redirect {
-      host        = "www.lastingpowerofattorney.service.gov.uk"
+      host        = "www.${local.dns_namespace_env}lastingpowerofattorney.service.gov.uk"
```

Comment:
With the `dns_namespace_env` var being populated with pre-prod URL will `www` addressing still work?

Reply:
i'm about to find out :)